packages/core/scripts/build/copyFiles.ts

```diff
@@ -27,7 +27,7 @@ interface CopyTask {
 const tasks: CopyTask[] = [
     { target: path.join('src', 'templates') },
     { target: path.join('src', 'test', 'shared', 'cloudformation', 'yaml') },
-    { target: path.join('src', 'test', 'codewhisperer', 'service', 'resources') },
+    { target: path.join('src', 'test', 'amazonqGumby') },
```

Comment:
why are we copying the whole test folder with .ts files? Can we only copy the resources inside?

Reply:
Yes we can, I missed the resources folder copy.